### ISSUE TITLE
explicitly setting adapter: require('sails-disk') breaks

### ISSUE THREAD

**Issue by tjwebb** [balderdashy/skipper#94]
```
 TypeError: undefined is not a function
    at buildOrNormalizeReceiver (/home/tjwebb/workspace/sails/node_modules/skipper/standalone/Upstream/build-or-normalize-receiver.js:65:24)
    at Upstream.upload (/home/tjwebb/workspace/sails/node_modules/skipper/standalone/Upstream/prototype.upload.js:117:22)
    at Object.module.exports.upload (/home/tjwebb/workspace/balderdashy/mcca/sails-hook-filemanager/api/controllers/FileController.js:19:24)
    at bound (/home/tjwebb/workspace/sails/node_modules/lodash/dist/lodash.js:729:21)
    at routeTargetFnWrapper (/home/tjwebb/workspace/sails/lib/router/bind.js:179:5)
    at callbacks (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:164:37)
    at param (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:138:11)
    at param (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:135:11)
    at pass (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:145:5)
    at nextRoute (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:100:7)
    at callbacks (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:167:11)
    at /home/tjwebb/workspace/sails/lib/router/bind.js:187:7
    at alwaysAllow (/home/tjwebb/workspace/sails/lib/hooks/policies/index.js:207:11)
    at routeTargetFnWrapper (/home/tjwebb/workspace/sails/lib/router/bind.js:179:5)
    at callbacks (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:164:37)
    at param (/home/tjwebb/workspace/sails/node_modules/express/lib/router/index.js:138:11) [TypeError: undefined is not a function]
```

**Comment by sailsbot** [comment]
Thanks for posting, @tjwebb.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
